examples/experimental/potree/src/app.tsx

```diff
@@ -0,0 +1,55 @@
+import React, {useState, useEffect} from 'react';
+import {createRoot} from 'react-dom/client';
+
+import Map from 'react-map-gl/maplibre';
+import maplibregl from 'maplibre-gl';
+import 'maplibre-gl/dist/maplibre-gl.css';
+
+import DeckGL from '@deck.gl/react';
+import {MapViewState} from '@deck.gl/core';
+import {PotreeSource} from '@loaders.gl/potree';
+
+import PotreeTile3DLayer from './potree-tile-3d-layer';
```

Comment:
Nit: Generally we try to void default exports
```suggestion
import {PotreeTile3DLayer} from './potree-tile-3d-layer';
```

Reply:
Done